rocks-lib/src/lua_package/mod.rs

```diff
@@ -8,12 +8,12 @@ pub use version::{PackageVersion, PackageVersionReq};
 mod outdated;
 mod version;
 
-pub struct LuaPackage {
+pub struct RemotePackage {
```

Comment:
This is the second major rename: a struct with only name and version information is merely a representation of a remote package somewhere on `luarocks.org`. This explains the rename, I think it conveys intent much more clearly.